src/components/RoomView/setupGame.jsx

```diff
@@ -38,7 +39,21 @@ function SetupGame() {
 axios.post("/api/25words", {selectedWordPackId})
  .then(response =>{return response})
  .then((result)=>{
-  dispatch(setWordsInGame(result.data))
+update(ref(database, 'rooms/' + roomId), {
+    words:result.data
+  })
+let createStore=ref(database, "rooms/" + roomId);
```

Comment:
Isn't this variable actually a ref to /rooms/roomId ? So a name more like roomRef ?
I'm not sure if this words but - maybe the logic is actually to do something like:

```js
let cardsRef = ref(database, `rooms/${roomId}/words`);
// If it doesn't exist yet, make it exist!
if (!snapshot.exists()) {
 // the stuff you have written
```

Reply:
yes we def can change it, I just used a random one to test out and did not spent too much thought on it